pkg/cmd/cli/cmd/deploy.go

```diff
@@ -250,6 +265,18 @@ func (c *retryDeploymentCommand) retry(config *deployapi.DeploymentConfig, out i
 		return fmt.Errorf(message)
 	}
 
+	// Delete the deployer pod as well as the deployment hooks pods, if any
+	pods, err := c.client.ListDeployerPodsFor(config.Namespace, deploymentName)
+	if err != nil {
+		return fmt.Errorf("Failed to list deployer/hook pods for the failed deployment: %v", err)
```

Comment:
The rest of the messages in this file do not have use this format and just use the depoyment number. I have made these messages consistent with the rest.